### DISCUSSION TITLE
Custom dashboard and chart refresh rate

### DISCUSSION THREAD

**Discussion by vobruba-martin** [netdata/netdata#12762] [Q&A]
Hi,

Is there any way how to set chart refresh rate in custom dashboard? Now I see that dygraph charts are refreshed every 2s and easypie charts every 12s. I'd like to set 1s refresh rate for all charts.

**Comment by ivangergo3** [comment]
Hi,

I think this is the link you are looking for
https://learn.netdata.cloud/docs/agent/web/gui/custom#changing-rates

**Reply by vobruba-martin** [reply]
No, this is related to chart units (for example if you want to change bytes to megabytes).

What I need is to refresh/redraw charts with the current data every second like the default dashboard does.
